scrapers_next/me/committies.py

```diff
@@ -0,0 +1,306 @@
+                pass
+
+            # 2) Back to top link
+            try:
+                XPath("div/h1").match_one(row)
+                back_to_top = True
+            except SelectorError:
+                pass
+
+            # 3) Membership Information
+            member_text = row.text_content()
+
+            # Committee names are cross-checked to make sure they match the
+            # names seen on a previous page. The scraper will break if an
+            # unexpected committee is seen.
+            if committee_name:
+                matched_committee = None
+                for c in committees:
+                    if c.name == committee_name:
+                        matched_committee = c
+                        committees.remove(c)
+                        break
+                if matched_committee:
+                    com = matched_committee
+                else:
+                    raise UnexpectedCommitteeName(com.name)
+
+            # Back to top link means that there is no more membership data
+            # for the current committee, and it can be yielded as long as
+            # there is at least one member assigned to it
+            elif back_to_top:
+                if com:
+                    if len(com.members) > 0:
+                        yield com
+                    else:
+                        self.logger.warning(f"No membership data found for: {com.name}")
+
+            # Member text is parsed to extract the name and role
+            # After that, it is added to the current committee
+            elif member_text:
+                if not com:
+                    raise MemberDataRowInUnexpectedLocation()
+                name, role = self.parse_member_title(member_text)
+                com.add_member(name=name, role=role)
+
+        # Any committees left in this list were not seen in the committee table
+        # and have no member info.
+        for com in committees:
+            self.logger.warning(f"No membership data found for: {com.name}")
+
+    def parse_member_title(self, text):
+        # Here's an example of the expected format of text
+        # "
+        #                    (Ranking Member)
+        #                                        Billy Bob Faulkingham  (R -
+        # Winter Harbor)
+        #
+        #
+        #
+        #                                 View Profile
+        # "
+
+        # Start by removing all newline characters and stripping whitespace
+        text = text.replace("\r\n", "").strip()
+
+        # The text now contains the following data separated with many spaces (" "):
+        # - Member role (optional)
+        # - Member name
+        # - Party-district
+        # - The string "View Profile"
+        # The individual blocks of data do not contain double spaces, so we can
+        # split by "  ".
+        data_blocks = text.split("  ")
+
+        # Then remove all empty strings from the list
+        while "" in data_blocks:
```

Comment:
`data_blocks = [b for b in data_blocks if b]` seems much cleaner.

Reply:
It is a lot cleaner this way, thank you. Fixed with https://github.com/openstates/openstates-scrapers/pull/4425/commits/3e348341945ab1fee61cda77bf06f7099b8e947f